src/main/java/com/cinema/Main.java

```diff
@@ -0,0 +1,98 @@
+package com.cinema;
+
+import com.cinema.facade.CinemaTicketsFacade;
+import com.cinema.model.hall.HallType;
+import io.vavr.control.Try;
+
+import java.util.Scanner;
+
+import static com.cinema.utils.DateTimeUtils.formatToLocalDate;
+import static com.cinema.utils.DateTimeUtils.formatToLocalTime;
+import static io.vavr.API.*;
+import static org.apache.commons.lang3.math.NumberUtils.toInt;
+
+public class Main {
+    public static final CinemaTicketsFacade facade = new CinemaTicketsFacade();
+
+    private static void printMenu() {
+        System.out.println("\n Select an option:\n" +
+                "1. Log in.\n" +
+                "2. Show all films in the cinema.\n" +
+                "3. Show all dates for the selected film.\n" +
+                "4. Show halls for the selected date.\n" +
+                "5. Show time of sessions in the selected hall.\n" +
+                "6. Show available places for the session with selected time.\n" +
+                "7. Book place.\n" +
+                "9. Exit.");
```

Comment:
Check the correctness of the numbering.

Reply:
Item 8 will be added for buying the ticket later.